Gemfile.lock

```diff
@@ -736,24 +755,32 @@ GEM
       rubocop-ast (>= 1.28.0, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 2.4.0, < 3.0)
-    rubocop-ast (1.28.1)
-      parser (>= 3.2.1.0)
-    rubocop-capybara (2.18.0)
+    rubocop-ast (1.31.3)
+      parser (>= 3.3.1.0)
+    rubocop-capybara (2.21.0)
+      rubocop (~> 1.41)
+    rubocop-factory_bot (2.26.0)
       rubocop (~> 1.41)
     rubocop-faker (1.1.0)
       faker (>= 2.12.0)
       rubocop (>= 0.82.0)
-    rubocop-rails (2.19.1)
+    rubocop-rails (2.25.1)
       activesupport (>= 4.2.0)
       rack (>= 1.1)
       rubocop (>= 1.33.0, < 2.0)
-    rubocop-rspec (2.20.0)
-      rubocop (~> 1.33)
+      rubocop-ast (>= 1.31.1, < 2.0)
+    rubocop-rspec (2.31.0)
+      rubocop (~> 1.40)
       rubocop-capybara (~> 2.17)
-    ruby-progressbar (1.11.0)
-    ruby-vips (2.1.4)
+      rubocop-factory_bot (~> 2.22)
+      rubocop-rspec_rails (~> 2.28)
+    rubocop-rspec_rails (2.29.0)
+      rubocop (~> 1.40)
```

Comment:
Those changes here are generating the error in the RSpec. Please take the older versions of the rubocop gems back